workspaces/optic-engine/tests/snapshots/domain_conflict_scenarios_generated__scenario_af20bd1d8e76cf9db0416175232b9b0276447f7148f3261f2d14b006aa7a0229__spectacle_endpoints_json.snap

```diff
@@ -133,10 +133,10 @@ expression: "&spec_projection.spectacle_endpoints_serializable()"
       }
     ],
     [
-      6,
-      4,
+      9,
```

Comment:
This is correct given how the response nodes are set in this fixture (there are two response nodes with `200` status codes).